### ISSUE TITLE
appendURL.lua fails to add url if previus file is over

### ISSUE THREAD

**Issue by joeonlist** [donmaiq/mpv-scripts#2]
Hi!
I noticed this issue:
- open mpv pseudo gui
- copy an URL to a video
- press "a" in the mpv empty gui
- the video starts and runs ok till it ends, so no issues
- after video is over, if mpv config option "idle=yes" is set, the gui stays open waiting for a new input file or url. So ok
- now if copy an other URL in my clipboard
- and press "a" in the gui
- nothing happens

Seems appendURL script is not recalled again as expected, when mpv window gui has already finished to play all previous videos and stays open in "idle mode".

Could this "bug" be fixed?
Thanks in advance for your work, anyway useful script!
Bye!  :)

**Comment by jonniek** [comment]
Hi,

Are you using windows? I can reproduce this bug on windows and it seems like subprocess doesn't allow powershell to be ran for some reason. I will investigate a bit further and maybe open an issue on mpv repository.

**Comment by joeonlist** [comment]
I'm on Linux Slackware 14.2.
```
$ mpv --version
mpv 0.29.1 Copyright © 2000-2018 mpv/MPlayer/mplayer2 projects
 built on Tue Sep 24 14:06:28 CEST 2019
ffmpeg library versions:
   libavutil       56.22.100
   libavcodec      58.35.100
   libavformat     58.20.100
   libswscale      5.3.100
   libavfilter     7.40.101
   libswresample   3.3.100
ffmpeg version: 4.1.4
```
Thanks for your reply!

**Comment by jonniek** [comment]
Fixed in c4e948ea3a5449cc959029961364e23ba14de748 by making subprocess uncancellable

seems like mpv has a feature where cancellable subprocess immidiately fails in idle.

**Comment by joeonlist** [comment]
I tried it, works fine now!
Good job jonniek!

Thanks a lot!  :D
